megatron/tokenizer/tokenizer.py

```diff
@@ -43,7 +43,8 @@ def build_tokenizer(args):
     elif args.tokenizer_type == "OpenGPTX-PretrainedHFTokenizer":
         tokenizer = PretrainedHFTokenizer.instantiate_from_file_or_name(model_file_or_name=args.tokenizer_model)
     elif args.tokenizer_type == "OpenGPTX-SPTokenizer":
-        tokenizer = SPTokenizer.instantiate_from_file_or_name(model_file_or_name=args.tokenizer_model)
+        #tokenizer = SPTokenizer.instantiate_from_file_or_name(model_file_or_name=args.tokenizer_model)
+        tokenizer = _GPTXSentencePieceTokenizer(args.tokenizer_model)
```

Comment:
ToDo: The _GPTXSentencePieceTokenizer is used for inference. Needs to be handled differently